backend/tsconfig.json

```diff
@@ -4,8 +4,11 @@
       "target": "ES6",
       "module": "CommonJS",
       "outDir": "./dist",
-      "rootDir": "./src",
       "strict": true,
       "skipLibCheck": true,
+      "paths": {
+        "@types": ["../types/index.ts"],
```

Comment:
I added a path alias so that we can use something like `import type { AnalysisResponse } from "@types";` instead of `import type { AnalysisResponse } from "../../../types";`